[MTM-61288] add KPI widget documentation to widget collection (#2651)
SoftwareAG/c8y-docs: content/cockpit/widgets-collection.md

  <a style="cursor:pointer;" onclick="location.hash = '#/group/<<group-id>>/dashboard/<<dashboard-id>>'">link to another dashboard</a><br />
```

### KPI {#kpi}

> 

Reviewer: @reey What is the "KPI" widget? Could you add 1-2 sentences to give the user an idea, what this widget is.

> The "KPI" widget visualizes a data point as a label. It is meant as a simple visualization for mainly KPIs (Key Performance Indicators).

Reviewer: ```suggestion
The "KPI" (Key Performance Indicators) widget visualizes a data point as a label. This allows you to create a visualization for main KPIs.
```

Author: ```suggestion
The "KPI" (Key Performance Indicators) widget visualizes a data point as a label. Such a data point could for example be the temperature of a device.
```

> The "KPI" (Key Performance Indicators) widget visualizes a data point as a label. Such a data point could for example be the temperature of a device.

Reviewer: ```suggestion
The "KPI" (Key Performance Indicators) widget visualizes a data point as a label, for example, a data point for the temperature of a device.
```

![KPI widget](/images/users-guide/cockpit/cockpit-widget-kpi.png)

> **Parameters to configure**

Author: ```
**Parameters to configure**
```

Is used in the whole file..

> On the left side, you can select the data point to be displayed.

Reviewer: ```suggestion
On the left side, select the data point you want to display.
```

> You have to select exactly one active datapoint to create the "KPI" widget.

Reviewer: @reey Why "exactly"? What happens when the user selects multiple data points?

Author: Then you can't save the configuration.

Reviewer: ```suggestion
You must select only one active datapoint to create the "KPI" widget. If you select multiple data points at once, you cannot save the configuration
```

> You must select only one active datapoint to create the "KPI" widget. If you select multiple data points at once, you cannot save the configuration

Reviewer: ```suggestion
You must select only one active datapoint to create the "KPI" widget. If you select multiple data points at once, you cannot save the configuration.
```

> On the right side, you can adjust how the datapoint is going to be displayed. This includes:

Reviewer: ```suggestion
On the right side, you can adjust how the data point is going to be displayed. This includes:
```

> * Icon: the icon to be displayed next to the data point

Reviewer: ```suggestion
* Icon: The icon to be displayed next to the data point
```
